scripts/installErizo.sh

```diff
@@ -53,6 +55,9 @@ install_erizo(){
   cd $ROOT/erizo
   ./generateProject.sh
   ./buildProject.sh $FAST_MAKE
+  if [ $DELETE_OBJECT_FILES == 'true' ]; then
```

Comment:
in other places we do `if [ "$DELETE_OBJECT_FILES" == "true" ];`, I didn't know it works like this too, great!

Reply:
Even if it works, I'll switch to `"` for consistency 🙂